src/test/java/com/wikia/webdriver/common/remote/context/CreatePostContext.java

```diff
@@ -0,0 +1,37 @@
+package com.wikia.webdriver.common.remote.context;
+
+import com.wikia.webdriver.common.remote.Discussions;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.remote.Discussions'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)